### PR TITLE
MGMT-17528: add ocp4.15 osImages on ACM 2.9

### PR DESCRIPTION
**Pull request by mihirlele**
## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [✔ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [✔ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [✔ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [✔ ] Title and description added to both, commit and PR.
- [✔ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [✔ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: deploy/podman/configmap.yml [hunk 1]
```diff
@@ -22,12 +22,13 @@ data:
...
.../dependencies/rhcos/4.14/4.14.0/rhcos-4.14.0-ppc64le-live.ppc64le.iso","version":"414.92.202309201615-0"},{"openshift_version":"4.14","cpu_architecture":"s390x","url":"https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.14/4.14.0/rhcos-4.14.0-s390x-live.s390x.iso","version":"414.92.202309201615-0"},{"openshift_version":"4.15","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-x86_64-live.x86_64.iso","version":"415.92.202402201450-0"},{"openshift_version":"4.15","cpu_architecture":"arm64","url":"https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-aarch64-live.aarch64.iso","version":"415.92.202402201450-0"},{"openshift_version":"4.15","cpu_architecture":"ppc64le","url":"https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-ppc64le-live.ppc64le.iso","version":"415.92.202402201450-0"},{"openshift_version":"4.15","cpu_architecture":"s390x","url":"https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-s390x-live.s390x.iso","version":"415.92.202402201450-0"}]'
```
Comment (danmanor) [inline deploy/podman/configmap.yml:25]: Redundant in podman deployment

File: openshift/template.yaml [hunk 2]
```diff
@@ -14,10 +14,10 @@ parameters:
...
...s/rhcos/4.14/4.14.0/rhcos-4.14.0-ppc64le-live.ppc64le.iso","version":"414.92.202309201615-0"},{"openshift_version":"4.14","cpu_architecture":"s390x","url":"https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.14/4.14.0/rhcos-4.14.0-s390x-live.s390x.iso","version":"414.92.202309201615-0"},{"openshift_version":"4.15","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-x86_64-live.x86_64.iso","version":"415.92.202402201450-0"},{"openshift_version":"4.15","cpu_architecture":"arm64","url":"https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-aarch64-live.aarch64.iso","version":"415.92.202402201450-0"},{"openshift_version":"4.15","cpu_architecture":"ppc64le","url":"https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-ppc64le-live.ppc64le.iso","version":"415.92.202402201450-0"},{"openshift_version":"4.15","cpu_architecture":"s390x","url":"https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.15/4.15.0/rhcos-4.15.0-s390x-live.s390x.iso","version":"415.92.202402201450-0"}]' # os images
```
Comment (danmanor) [inline openshift/template.yaml:17]: Redundant changes in this file. it is for SaaS deployment

**Review by danmanor — CHANGES_REQUESTED** [review]

**Review by mihirlele — COMMENTED** [review]
Made the changes as per the comments

**Comment by mihirlele** [comment]
@danmanor I have rolled back the changes in the files that you said were not required. Please check and let me know if anything else is required to be done.

**Comment by danmanor** [comment]
@mihirlele please run `skipper make generate` to pass the code generation test

**Comment by mihirlele** [comment]
obsolete due to https://github.com/openshift/assisted-service/pull/6201. Closing

**Pull request closed without merging**